bin/start-in-linux

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+
+declare minikube=$(which minikube);
+declare kubectl=$(which kubectl);
+declare workdir=$(cd $(dirname $0)/.. && pwd);
+
+
+start_minikube() {
+	# set the env
+	export MINIKUBE_WANTUPDATENOTIFICATION=false;
+	export MINIKUBE_WANTREPORTERRORPROMPT=false;
+	export MINIKUBE_HOME=$HOME;
+	export CHANGE_MINIKUBE_NONE_USER=true;
+	mkdir $HOME/.kube || true;
+	touch $HOME/.kube/config;
+
+	echo "[*] Starting minikube ...";
+	export KUBECONFIG=$HOME/.kube/config;
+	sudo -E ${minikube} start --vm-driver=none;
```

Comment:
I don't see why this can not be generalized to work on OSX.

Several differences: 
1. On OS X we must not use `sudo` here
2. `--vm-driver` must be either skipped to use the default driver of switched to 'docker-machine-driver-xhyve'